keyboards/kinesis/kint36/kint36.c

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2020 QMK
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "kint36.h"
+
+void matrix_init_kb(void) {
+    led_init_ports();
+    matrix_init_user();
+}
+
+#define ledTurnOff writePinHigh
+#define ledTurnOn writePinLow
+
+#define LED_KEYPAD E26
+#define LED_SCROLL_LOCK A5
+#define LED_NUM_LOCK A14
+#define LED_CAPS_LOCK C7
```

Comment:
https://docs.qmk.fm/#/feature_led_indicators

Reply:
Done, thanks for the hint!